### PR TITLE
JBIDE-11436 use license feature instead of bo...
...ilerplate license in feature.properties and license.html files

### PR DESCRIPTION
**Pull request by nickboldt**

### CODE REVIEW

**Comment by maxandersen** [comment]
this PR should not be merged in before https://github.com/jbosstools/jbosstools-base/pull/420 is ready

**Comment by nickboldt** [comment]
https://github.com/jbosstools/jbosstools-base/pull/420 was merged. This should now be safe to merge after building locally to verify.

File: features/org.jboss.tools.arquillian.feature/feature.xml [hunk 1]
```diff
@@ -14,7 +16,7 @@
       %copyright
    </copyright>
 
-   <license url="%licenseURL">
+   <license>
```
Comment (fbricon) [inline features/org.jboss.tools.arquillian.feature/feature.xml]: shouldn't you remove the whole license node? given %license can not be resolved anymore...

**Comment by nickboldt** [comment]
No, %license will be resolved once the contents from the upstream license feature are merged in.

Try it and see!

File: features/org.jboss.tools.arquillian.feature/feature.xml (hunk 1 above)
Comment (nickboldt) [inline features/org.jboss.tools.arquillian.feature/feature.xml]: No. the license.html is generated by the upstream license feature. I had previously thought we should remove this, but we in fact have to keep the license url="%licenseURL" part, so it points to license.html. The devdoc was updated to reflect this correction.

https://github.com/jbosstools/jbosstools-devdoc/commit/c3037f6632d550d834505822f912c4033408d205

**Pull request merged**